src/ClientConfig.cc

```diff
@@ -325,12 +326,21 @@ bool ClientConfig::LoadConfig(const std::string &_file)
           {
             // Sanity check: Make sure that the server is not already stored.
             bool repeated = false;
-            for (auto const savedServer : this->Servers())
+            for (auto & savedServer : this->Servers())
             {
               if (savedServer.Url().Str() == serverURL)
               {
-                ignwarn << "URL [" << serverURL << "] already exists. "
-                       << "Ignoring server" << std::endl;
+                if (!privateToken.empty())
+                {
+                  ignerr << "Setted key for " << serverURL << " server."
```

Comment:
This shouldn't be an error, right?

Also, maybe saying "private token" would be better and match the parameter

```suggestion
                  ignmsg << "Set private token for " << serverURL << " server."
```

The same goes for line 355